src/common/components/form/validationUtils.tsx

```diff
@@ -18,9 +15,7 @@ const validateRequire = (value: any, customMessage?: string) => {
 };
 
 /** validateBirthdate()
- * Validate user input child birthdate.
- * This app only target recently born child, so we support child which born after 2019 only.
- * Input time which come from future is not accepted
+ * Validate input birth hdate. Dates in the future are not valid.
  * @param value Input value.
  */
 const validateBirthdate = (value: string | number) => {
```

Comment:
rename function

Reply:
I thought we agreed to leave it as birthdate (since the validation includes born-after-date?)

Reply:
No, i still think it should be validateDate. But its fine

Reply:
I renamed it now for your pleasure.